mobify/platform-scaffold
# WEB-1526: Fix missing account-dashboard chunk

**JIRA**: [WEB-1526](https://mobify.atlassian.net/browse/WEB-1526)

## Changes
- Update webpackChunkName so the account dashboard file downloads

| Before |
| --- |
| ![screen shot 2017-09-13 at 5 19 17 pm](https://user-images.githubusercontent.com/734535/30406601-b80fffe6-98a7-11e7-9fd8-dfdcbe5f98f0.png) |


## Todos
- ~~(if applicable) analytics events instrumented to measure impact of change~~
- ~~Add a high-level description of your changes to the CHANGELOG.md, including a link to the PR with your changes~~

## How to test-drive this PR
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- **Verify** that the error in the above screenshot is not present
